client/scripts/startup.js

```diff
@@ -1,13 +1,24 @@
 /* eslint-disable global-require */
-function start() {
-  require('../app/bundles/Course/LessonPlan/startup');
+/* eslint-disable import/no-dynamic-require */
+
+function loadCurrentModule() {
+  const modulePath = document.head.querySelector("[name='server-context']")
+                       .getAttribute('data-controller-name');
+
+  try {
+    require(`../app/bundles/${modulePath}`);
```

Comment:
I am following our existing convention and put exiting `helpers` to `client/lib/helpers`, maybe the folders can be further renamed or better structured.